File: javaApp/src/main/java/io/github/rebirthlee/spinnermvvm/java/BindingAdapterHelper.java
package io.github.rebirthlee.spinnermvvm.java;

import android.widget.ImageView;

import com.bumptech.glide.Glide;

import androidx.databinding.BindingAdapter;

import io.github.rebirthlee.spinnermvvm.java.GlideApp;

public class BindingAdapterHelper {
    @BindingAdapter({"imgUrl"})
    public static void loadProfileImage(ImageView view, String url) {
        GlideApp.with(view)
                .load(url)
                .circleCrop()
                .into(view);
    }
}

File: kotlinApp/src/main/java/io/github/rebirthlee/spinnermvvm/kotlin/BindingAdapterHelper.kt
package io.github.rebirthlee.spinnermvvm.kotlin

import android.widget.ImageView
import androidx.databinding.BindingAdapter

object BindingAdapterHelper {
    @JvmStatic
    @BindingAdapter("imgUrl")
    fun loadProfileImage(view: ImageView, url: String) {
        GlideApp.with(view)
                .load(url)
                .circleCrop()
                .into(view)
    }
}
